applications/FluidDynamicsApplication/custom_utilities/drag_utilities.cpp

```diff
@@ -32,7 +32,7 @@ namespace Kratos
         // Sum the reactions in the model part of interest.
         // Note that the reactions are assumed to be already computed.
         VariableUtils variable_utils;
-        array_1d<double, 3> drag_force = variable_utils.SumHistoricalNodeVectorVariable(REACTION, rModelPart, 0);
+        auto drag_force = variable_utils.SumHistoricalVariable<array_1d<double,3>>(REACTION, rModelPart, 0);
```

Comment:
do you actually need to specify the template parameter? i would have expected that it detects it alone

Reply:
I tried and it can't. :disappointed: